int/api/massa/addresses.go

```diff
@@ -0,0 +1,61 @@
+package massa
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/thyra/api/swagger/server/models"
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/pkg/node"
+)
+
+func AddressesHandler(params operations.MassaGetAddressesParams) middleware.Responder {
+	client := node.NewDefaultClient()
+
+	addressesDetails, err := node.Addresses(client, params.Body.Addresses)
+	if err != nil {
+		return operations.NewMassaGetAddressesInternalServerError().
+			WithPayload(
+				&models.Error{
+					Code:    "get_Addresses error",
+					Message: "Error : Cannot get result from Address: " + err.Error(),
```

Comment:
This error is useless.
```suggestion
					Message: fmt.Sprintf("while getting details of addresses %v: %s\n", params.Body.Addresses, err),
```